apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/MapLikeToContainInAnyOrderCreatorSamples.kt

```diff
@@ -6,4 +6,37 @@ import kotlin.test.Test
 
 class MapLikeToContainInAnyOrderCreatorSamples {
 
+    @Test
+    fun entriesOf() {
+        expect(
+            mapOf(
+                1 to "a",
+                2 to "b",
+            )
+        )
+            .toContain
+            .inAnyOrder
+            .entriesOf(
+                mapOf(
+                    2 to "b",
+                )
+            )
```

Comment:
I suggest you use a more compact format, how about:
```suggestion
        expect(mapOf(1 to "a", 2 to "b")).toContain.inAnyOrder.entriesOf(
            mapOf(2 to "b")
        )
```

Reply:
Thanks for the review! Reducing the space makes sense!